test/helpers/createSearchClient.ts

```diff
@@ -0,0 +1,50 @@
+interface Options {
+  [key: string]: any;
+}
+
+/**
+ * Generates search client fixture for testing components that rely on hooks
+ * from the Algolia instantsearch library.
+ *
+ * Example usage:
+ * ```
+ * const searchClient = createSearchClient({
+ *   facets: {
+ *     categories: {
+ *       "Arts, Culture & Identity": 54,
+ *       Education: 28,
+ *     },
+ *   },
+ * });
+ *
+ * render(
+ *   <InstantSearch
+ *     searchClient={searchClient}
+ *     indexName="fake_test_search_index"
+ *   >
+ *     <BrowseRefinementList attribute="categories" />
+ *   </InstantSearch>
+ * );
+ * ```
+ *
+ * @param options Additional customizations of the search response
+ * @returns
+ */
+export function createSearchClient(options: Options) {
+  return {
+    search: (requests: any) =>
```

Comment:
**question (if-minor):** In the interest of minimizing lint warnings, can we change these 2 'any' types in this file or can we lint ignore this file?

Reply:
Oops yeah, forgot to add ignore statements. I was going to clean up all the warnings in a separate PR so I can do it there.